apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -46,7 +46,7 @@ ${shapeId} {
       ${sh.languageIn} ("en" "de" "fr" "it") ;
       ${sh.uniqueLang} true ;
       ${sh.order} 10 ;
-      ${sh.description} "Provide a publishable title describing the cube. It is mandatory to define the language." ;
+      ${sh.description} "A publishable title describing the cube. You can add multiple entries for different [languages](https://gitlab.ldbar.ch/bafu/cube-creator-documentation/-/wikis/(2)-Meta-Data-and-Manual-Validation-in-Cube-Preview#languages). It is mandatory to define the language for each entry and advised to not have multiple entries for the same language." ;
```

Comment:
It is, in fact, forbidden to have multiple values in any given language

Reply:
nice to know.. so i will remove it here

Reply:
:+1:

Notice `sh:uniqueLang` just above.